streaming/src/main/scala/org/apache/spark/streaming/scheduler/Job.scala

```diff
@@ -25,15 +25,43 @@ import scala.util.Try
  */
 private[streaming]
 class Job(val time: Time, func: () => _) {
-  var id: String = _
-  var result: Try[_] = null
+  private var _id: String = _
+  private var _outputOpId: Int = _
```

Comment:
Could you define what is `id` and `outputId` (especially that output id is unique within the jobset)